util/mdbook-reggen

```diff
@@ -95,7 +95,7 @@ def main() -> None:
 
             name = reg[1] + "." + reg[2] if len(reg) == 3 else reg[1]
 
-            return "[`{}`]({}#Reg_{})".format(name, path, reg[1].lower())
```

Comment:
Is this something that can be squashed into the commit that introduces the reggen preprocessor?